compose/cli/main.py

```diff
@@ -633,11 +633,13 @@ def pull(self, options):
         Options:
             --ignore-pull-failures  Pull what it can and ignores images with pull failures.
             --parallel              Pull multiple images in parallel.
+            --quiet                 Pull without printing progress information
```

Comment:
I think `--no-progress` might be a better, more descriptive name for this option. `--quiet` could mean a number of different things.